extension.js

```diff
@@ -50,11 +52,11 @@ function generateDescBox(editor, login) {
             editBuilder.insert(
                 new vscode.Position(0, 0),
                 `${chars[0]}\n\
-${chars[1]} ETNA PROJECT, ${convertDate(Date())} by ${login}\n\
-${chars[1]} ${folder.uri.fsPath}\n\
-${chars[1]} File description:\n\
-${chars[1]}      ${desc}\n\
-${chars[2]}\n\n`
+                ${chars[1]} ETNA PROJECT, ${convertDate(Date())} by ${login}\n\
```

Comment:
Si ils était mal indenté c'était pour une raison !
```suggestion
                ${chars[1]} ETNA PROJECT, ${convertDate(Date())} by ${login}\n\
```